packages/alpine/build.sh

```diff
@@ -56,4 +56,3 @@ termux_step_create_debscripts() {
 	echo "touch \$HOME/.pine-passfile" >> postinst
 	echo "fi" >> postinst
 }
-
```

Comment:
As mentioned my Apollo, a single newline at the end of the file is the POSIX standard, so these shouldn't be removed.

Reply:
There is a newline like sylirre mentioned but it really depends on the editor to show it, for example gedit does show it.
The file libandroid-execinfo/LICENSE does not have a newline at the end and that's somebody need to fix it.
Technically we should check U+000A which is now incorporated into the linter.